.github/workflows/auto-tag.yml

```diff
@@ -1,8 +1,8 @@
 name: Monthly Tagging
 on:
   schedule:
-    # Run every month on the 1st day at 08:15 AM.
-    - cron: '15 8 * 1 *'
+    # Run every month on the 3rd day at 08:15 AM.
+    - cron: '15 8 3 * *'
```

Comment:
Changed the schedule from the 1st of the month to 3rd of the month, as I hope this fix gets approved and merged, before this possible next schedule and we see it in action.
And as we are supposed to provide a tagged version for OTel collector release, tagging should happen rather soonish.